### PR TITLE
fix: The type of the output `cloudwatch_log_group_name` should be a string and not a list of string

### PR DESCRIPTION
**Pull request by paul-pop**
# PR o'clock

## Description

Fixes https://github.com/terraform-aws-modules/terraform-aws-eks/issues/1060

### Checklist

- [x] CI tests are passing
- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -48,7 +48,7 @@ output "cluster_iam_role_arn" {
 
 output "cluster_oidc_issuer_url" {
   description = "The URL on the EKS cluster OIDC Issuer"
-  value       = flatten(concat(aws_eks_cluster.this[*].identity[*].oidc.0.issuer, [""]))[0]
+  value       = element(flatten(concat(aws_eks_cluster.this[*].identity[*].oidc.0.issuer, list(""))), 0)
```
Comment (barryib) [inline outputs.tf:51]: This change (+ [`kubeconfig_filename`](https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1061/files#diff-de6c47c2496bd028a84d55ab12d8a4f90174ebfb6544b8b5c7b07a7ee4f27ec7R71) and [`oidc_provider_arn`](https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1061/files#diff-de6c47c2496bd028a84d55ab12d8a4f90174ebfb6544b8b5c7b07a7ee4f27ec7R76)) sounds not related to your bug. Can you please revert those changes.

File: variables.tf [hunk 2]
```diff
@@ -312,7 +312,7 @@ variable "create_eks" {
 }
 
 variable "node_groups_defaults" {
-  description = "Map of values to be applied to all node groups. See `node_groups` module's documentaton for more details"
+  description = "Map of values to be applied to all node groups. See `node_groups` module's documentation for more details"
```
Comment (barryib) [inline variables.tf:315]: Nice catch ^^

**Review by barryib — CHANGES_REQUESTED** [review]

File: outputs.tf (hunk 1 above)
Comment (paul-pop) [reply outputs.tf]: Fair enough, I added them so all outputs are consistent. I reverted this now.

**Comment by paul-pop** [comment]
@barryib happy to merge this?

**Review by barryib — APPROVED** [review]

**Comment by barryib** [comment]
Thanks @paul-pop for your contribution.

**Pull request merged**